tianshou/data/batch.py

```diff
@@ -29,6 +29,30 @@ def _is_batch_set(data: Any) -> bool:
     return False
 
 
+def is_scalar_value(value: Any) -> bool:
+    # check if the value is a scalar
+    # 1. python bool object, number object: isinstance(value, Number)
+    # 2. numpy scalar: isinstance(value, np.generic)
+    # 3. python object rather than dict / Batch / tensor
+    # the check of dict / Batch is omitted because this only checks a value.
+    # a dict / Batch will eventually check their values
+    is_scalar = isinstance(value, Number)
+    is_scalar = is_scalar or isinstance(value, np.generic)
+    is_scalar = is_scalar or (isinstance(value, (np.ndarray, torch.Tensor))
+                              and not value.shape)
+    return is_scalar
+
+
+def is_number_value(value: Any) -> bool:
```

Comment:
same

Reply:
And I not to simply call it `is_number` ?

Reply:
Fixed in 44034b6 .